src/jackdaw/streams.clj

```diff
@@ -312,13 +314,23 @@
   []
   (interop/streams-builder))
 
-(defn kafka-streams
-  "Makes a Kafka Streams object."
-  ([builder opts]
-   (let [props (java.util.Properties.)]
-     (.putAll props opts)
-     (KafkaStreams. ^Topology (.build ^StreamsBuilder (streams-builder* builder))
-                    ^java.util.Properties props))))
+(defn ^Topology topology
```

Comment:
This block decouples the creation of the streams app from the creation of the underlying Topology. This is particularly useful for drilling down into the Topology for doing things like accessing the [description](https://kafka.apache.org/23/javadoc/index.html?org/apache/kafka/streams/Topology.html) as well as opening up the way to doing datafy/nav using the active system's topology.